input/pagecontent/3-2-1-syncerror.md

```diff
@@ -4,51 +4,38 @@ eventMaturity | [2 - Tested](3-1-2-eventmaturitymodel.html)
 
 ### Workflow
 
-A synchronization error has been detected. Inform subscribed clients. 
+A synchronization error has been detected and the condition is indicated to subscribed clients. 
 
 Unlike most of FHIRcast events, `syncerror` is an infrastructural event and does not follow the `FHIR-resource`-`[open|close]` syntax and is directly referenced in the [underlying specification](2_Specification.html).
 
+A `syncerror` is sent by a subscribed application when:
+1. It responds to a context change event with a 202 indicating the context change is accepted but has not yet occurred, then the application decides to refuse the context (see: [`Event Notification Response`](2-5-EventNotification.html#event-notification-response)) - the `severity` of the `operationoutcome` resource in the `syncerror` SHALL be `warning`
```

Comment:
It responds to a context change event with a 202 indicating the context change is accepted but has not yet occurred, **and** the application decides to refuse the context (see: [`Event Notification Response`](2-5-EventNotification.html#event-notification-response)) - the `severity` of the `operationoutcome` resource in the `syncerror` SHALL be `warning`

Reply:
Eric to accept.

Reply:
Completed